src/config/index.ts

```diff
@@ -19,13 +19,14 @@ export type ContactType = {
   contact_type: string;
   contact_friendly?: string;
   user_role: string[];
-  username_from_place: boolean;
+  username_from_place?: boolean;
```

Comment:
why not require these to have explicit values? seems cleaner to be explicit instead of having to worry about defaults